Akanksha.bathini/update usage page (#10767)
DataDog/documentation: content/en/account_management/multi_organization.md

The parent-organization can view the total and billable usage of all their organizations (child and parent organizations) by hovering over their username at the bottom left and then navigating to: `Plan & Usage`--> `Multi-Org Usage`.

The Multi-Org Usage page shows the aggregate usage of the parent-organization and all its child-organizations. There are two tabs on the Multi-Org Usage page:

* Overall
* Individual Organization

### Overall usage

This tab contains a Month-to-Date Total Usage section and an Overall Usage section.

The Month-to-Date Total Usage section summarizes your month-to-date usage of hosts, containers, custom metrics, and any other part of the platform you’ve used during the month, across your parent-organization and all its child-organizations.

{{< img src="account_management/multi-org-v2.png" alt="Month-to-Date Usage" >}}

> Most accounts by default will be able to view "Billable" usage, which only shows usage that will contribute to your final bill. This view also breaks out on-demand usage above your commitments and allocations. Switching to the "All" view will show you all usage, including non-billable usage such as product trials.

Reviewer: ```suggestion
Most accounts by default can view "Billable" usage, which shows usage that contributes to your final bill. This view also breaks out on-demand usage above your commitments and allocations. The "All" view shows you all usage, including non-billable usage such as product trials.
```

{{< img src="account_management/multi-org-v2-trends.png" alt="Overall Usage Long-term trends" >}}

Both the Month-to-Date Total Usage section and the Overall Usage section can be filtered by clicking on product specific sub-tabs. In the "Log Management" sub-tab, you can view the Logs Usage by Index table, which displays your month-to-date and last month's indexed log usage by:

* Index name
* Organization
* Retention period in days
* Indexed log count broken down between live and rehydrated logs
* The index's contribution percentage to the overall indexed log usage

This data can be downloaded as a CSV file.

{{< img src="account_management/multi-org-v2-logs-by-index.png" alt="Multi-org Logs Usage by Index" >}}

### Individual Organization usage

In the Individual Organization usage tab, you are able to view the usage of your child organizations in either absolute units or as a percent of total usage.

{{< img src="account_management/multi-org-percent-billable-v2.png" alt="Individual Percent Usage" >}}

> The default view is the "Billable" view, which only shows usage that contributes to your final bill. This view removes child organizations that are not billable such as trial organizations in addition to other adjustments that will provide a more accurate summary of what drives your bill. If you switch to the "All" view, you will be able to see the unadjusted, raw usage of your parent-organization and all child-organizations. Both views can be downloaded as a CSV file.

Reviewer: ```suggestion
The default view is the "Billable" view, which shows usage that contributes to your final bill. This view removes child organizations that are not billable such as trial organizations, and other adjustments that provide a more accurate summary of what drives your bill. Switch to the "All" view to see the unadjusted, raw usage of your parent-organization and all child-organizations. Both views can be downloaded as a CSV file.
```
